src/Magento/FunctionalTestingFramework/DataGenerator/Handlers/SecretStorage/VaultStorage.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\FunctionalTestingFramework\DataGenerator\Handlers\SecretStorage;
+
+use Magento\FunctionalTestingFramework\Config\MftfApplicationConfig;
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+use Magento\FunctionalTestingFramework\Util\Logger\LoggingUtil;
+use Vault\Client;
+use VaultTransports\Guzzle6Transport;
+
+class VaultStorage extends BaseStorage
+{
+    /**
+     * Mftf project path
+     */
+    const MFTF_PATH = '/mftf';
+    /**
+     * Vault kv version 2 data
+     */
+    const KV2_DATA = 'data';
+
+    /**
+     * Default vault token file
+     */
+    const TOKEN_FILE = '.vault-token';
+    /**
```

Comment:
Add space between const declaration and comment line, do this for all of your changes.

Reply:
Fixed